app/lib/server/runner.ts

```diff
@@ -194,6 +227,30 @@ export const run = async (ctx: Context, space: SpaceDID, dialogs: Set<bigint>, p
 }
 
 
+const syncDialogEntity =  async (client: TelegramClient, entityId: string) => {
+  console.log(`Entity not found for ID: ${entityId}, trying to sync...`)
+  for await (const dialog of client.iterDialogs()) {
+    if (dialog?.entity?.id.toString() === entityId) {
+      break
+    }
+  }
+}
+
+const callWithDialogsSync = async <T>(
+    fn: () => Promise<T>,
+    client: TelegramClient,
+    entityId: string
+): Promise<T> => {
+  try {
+    return await fn()
+  } catch (error) {
+    if (error instanceof Error && error.message?.includes("Could not find the input entity for ")) {
```

Comment:
Again, depending on the error message is super brittle...is there nothing better we can do?